### PR TITLE
refactor: fs async methods to Sync 

### PR DESCRIPTION
**Pull request by vincanger**

### CODE REVIEW

File: generateHashArrayFile.js [hunk 1]
```diff
@@ -36,76 +36,66 @@ const generateLeavesHashArray = async () => {
...
+    console.log(`Leaf Count: ${leafCount}`);
+    console.log(`Depth ${treeDepth}`);
 }
 
-generateLeavesHashArray()
-    .then(generateMerkleLeaves()
-        .then(generateMerkleTree()
-            .then(generateMerkleRoot()
-                .then(runLogs()))));
+const runThisShit = () => {
```
Comment (joselvelez) [inline generateHashArrayFile.js]: suggest renaming

File: generateHashArrayFile.js [hunk 2]
```diff
@@ -36,76 +36,66 @@ const generateLeavesHashArray = async () => {
...
+    generateLeavesHashArray();
+    generateMerkleLeaves();
+    generateMerkleTree();
+    generateMerkleRoot();
+    runLogs();
+    console.log('...Everything is Merkled');
+} 
+
+runThisShit();
```
Comment (joselvelez) [inline generateHashArrayFile.js]: suggest renaming

**Review by joselvelez — CHANGES_REQUESTED** [review]

File: generateHashArrayFile.js (hunk 1 above)
Comment (vincanger) [reply generateHashArrayFile.js]: I refuse!

File: generateHashArrayFile.js (hunk 2 above)
Comment (vincanger) [reply generateHashArrayFile.js]: ok, fine

**Review by joselvelez — APPROVED** [review]

**Pull request merged**